src/redux/reducers/appConfig.ts

```diff
@@ -147,11 +149,20 @@ export const configSlice = createSlice({
       electronStore.set('appConfig.folderReadsMaxDepth', action.payload);
       state.folderReadsMaxDepth = action.payload;
     },
+    updateClusterNamespaces: (state: Draft<AppConfig>, action: PayloadAction<ClusterAccess[]>) => {
+      if (!state.projectConfig) {
+        return;
+      }
+      electronStore.set('appConfig.settings.clusterNamespaces', action.payload);
+      state.projectConfig.clusterAccess = action.payload;
+    },
     updateK8sVersion: (state: Draft<AppConfig>, action: PayloadAction<string>) => {
       electronStore.set('appConfig.k8sVersion', action.payload);
       state.k8sVersion = action.payload;
     },
     setCurrentContext: (state: Draft<AppConfig>, action: PayloadAction<string>) => {
+      execSync(`kubectl config use-context ${action.payload}`);
```

Comment:
I'm wondering if we should check if this fails for some reason and don't change the context if that happens?

Reply:
Yes, that seems like a good idea